output/plugins/common/xml/common.cppcode

```diff
@@ -33,6 +33,27 @@ Written by
 	    #ifnotequal $default "0" @{
 		    $name->SetDefault(); #nl
 	    @}
+
+			#ifnotnull $bitmap
+			@{ #nl $name->SetBitmap( $bitmap ); @}
```

Comment:
Can't this be removed as `$bitmap` is already passed to the constructor?

Reply:
This hunk is inside wxButton, that doesn't get the bitmap from the constructor

Reply:
It still gets unnecessarily added for `wxBitmapButton` (because it inherits from `wxButton`?) though.

Reply:
Interesting, its even worse than that. The settings templates of all inherited classes get applied and because the bitmaps get set in both this results in setting all the other bitmaps twice. Nice catch, i didn't discover this in all these years i was using that version.

I'm going to fix that, plus im going to fix #390, the markup isn't stored in XRC.